spec/features/navigation_spec.rb

```diff
@@ -32,6 +56,70 @@
         expect(page).to have_content("Cart: 0")
       end
 
+      visit '/cart'
+
+      within 'nav' do
+        expect(page).to have_content("Cart: 0")
+      end
+
+      visit '/login'
+
+      within 'nav' do
+        expect(page).to have_content("Cart: 0")
+      end
+
+      visit '/register'
+
+      within 'nav' do
+        expect(page).to have_content("Cart: 0")
+      end
```

Comment:
Nailed it! That's some solid testing.

Reply:
![](https://media.giphy.com/media/HzjNv9lL25fNe/giphy.gif)